elastic/kibana
# [Watcher] Use fixed_interval instead of interval

From `8.x` ES no longer support the `interval` parameters for the date_histogram aggregation. It is now required to use the more precise `fixed_interval` or `calendar_interval`. 
Looking at how the code is generating currently the interval for the threshold watch I decided that `fixed_interval` was the more appropriate.

## How to test

* Navigate to the watcher app
* Create a new threshold alert
* Fill the required fields (name, index and time field). You should see the visualise chart underneath


<img width="917" alt="Screenshot 2021-09-30 at 12 19 26" src="https://user-images.githubusercontent.com/2854616/135446031-6f2d9caa-8ad7-40c1-a9cd-7d4fe020ca99.png">

## Tests

As part of this PR I've removed all use of `nextTick()` in the watcher jest tests (https://github.com/elastic/kibana/pull/113527/commits/03e40c884f7130ac739ed13076c87984b310cc24)

Fixes https://github.com/elastic/kibana/issues/108754
